compiler-rt/lib/xray/xray_tsc.h

```diff
@@ -96,7 +96,13 @@ namespace __xray {
 inline bool probeRequiredCPUFeatures() XRAY_NEVER_INSTRUMENT { return true; }
 
 ALWAYS_INLINE uint64_t readTSC(uint8_t &CPU) XRAY_NEVER_INSTRUMENT {
+#if defined(__clang__)
   return __builtin_readcyclecounter();
+#else
+  uint64_t Cycles;
+  asm volatile("stckf %0" : /* No output. */ : "Q"(Cycles) : "cc");
```

Comment:
remove the unnecessary comment `/* No output. */`. In the majority of places we assume that the user is aware of the `: : ` syntax